app/src/components/sections/SingleTeamView/forms/ApproveUsersEIP712Form.vue

```diff
@@ -251,7 +253,10 @@ const submitApprove = () => {
   emit('approveUser', {
     approvedAddress: formData.value[0].address,
     budgetType: budgetLimitType.value,
-    value: limitValue.value,
+    value:
+      budgetLimitType.value === 0
+        ? Number(limitValue.value)
+        : web3Library.parseEther(limitValue.value),
```

Comment:
I suggest you use viem's parseEther instead

Reply:
I'll just let it be for now, once I get the functionality right I'll refactor. It also has implications on the tests and I had written the tests for the smart contract using Ethers as well so it will affect a number of things.